addons/survey/static/src/js/survey_quick_access.js

```diff
@@ -24,8 +25,11 @@ publicWidget.registry.SurveyQuickAccessWidget = publicWidget.Widget.extend({
             if (!self.readonly) {
                 $(document).on('keypress', self._onKeyPress.bind(self));
             }
-
-            self.$('input').focus();
+            // Put the cursor after the input value
+            var sessionCode = self.$('input#session_code').val();
+            self.$('input#session_code').focus();
+            self.$('input#session_code').val('');
+            self.$('input#session_code').val(sessionCode);         
```

Comment:
I think what Luc meant in the specifications is that the focus in the _empty_ input looks strange:

![image](https://user-images.githubusercontent.com/41282536/164010107-c6dbd783-30ac-4d43-9aa6-d8d127d35435.png)

Maybe we should simplify and avoid focus?
It's a bit annoying though because it's more practical when you have a direct focus to be able to type.
Since there is literally nothing else to do on that page.

Let's keep the current code with a simple focus, and we will re-discuss this with him at the testing phase if he doesn't like it.

Reply:
Not sure what was meant with "with a simple focus". I removed the autofocus for the moment.

In fact, to fix the strange empty focus layout, I added the recopy of the session code in the input when using a link containing a code with an error. Previously it simply redirected to "/s". That way, a user should never land on a quick survey access page with an empty input, since normally a user should receive a link containing a session code.

That's why I added this piece of code that puts the cursor at the end of the input when focusing, since most of the time the input will have a default value.

Reply:
> I added the recopy of the session code in the input when using a link containing a code with an error.

Yes that's a good idea, I found that a bit frustrating when testing myself :+1: